src/kit/Toggle.tsx

```diff
@@ -10,9 +10,18 @@ interface Props {
 }
 
 const Toggle: React.FC<Props> = ({ checked = false, label, onChange }: Props) => {
+  const [toggled, setToggled] = useState(checked);
+  useEffect(() => {
+    setToggled(checked);
+  }, [checked, setToggled]);
```

Comment:
nit: setToggled doesn't need to be in the dep array

Reply:
✅